Userland/Libraries/LibWeb/HTML/HTMLInputElement.cpp

```diff
@@ -415,13 +408,7 @@ void HTMLInputElement::did_pick_color(Optional<Color> picked_color, ColorPickerU
         update_color_well_element();
 
         // the user agent must queue an element task on the user interaction task source
```

Comment:
This partial sentence left on its own now seems quite awkward

Reply:
Duplicate comments removed.